### ISSUE TITLE
Finetune ALBERT using pretained model

### ISSUE THREAD

**Issue by astrongstorm** [google-research/ALBERT#87]
Hi,

Thanks very much for releasing the pre-trained code for ALBERT. I have downloaded THhub model from https://tfhub.dev/google/albert_base/1 , However, I found the accuracy of CoLA and MNLI seems to indicate, this is model does not contain pre-trained weight fro the ALBERT. 
I have tried:

python3 -m run_classifier_sp --data_dir=../../DataSet/CoLA/ --task_name=cola —-output_dir=testing_sp_modifying_model_setting_in_main_file --vocab_file=vocab.txt --albert_config_file=2/assets/albert_config.json --do_train=False --do_eval=True --max_seq_length=128  --train_batch_size=32  --learning_rate=2e-05 --num_train_epochs=3.0

with do_train set to False, getting the accuracy of around 40%,
however, with do_train set to True, I am able to get the accuracy of 70%

Besides, after downloading the weight from TH_Hub, I can only find the .json file, but without the .ckpt file from the folder.

I am wondering, how can we find the per-trained weight, the .ckpt file for ALBERT?

**Comment by 0x0539** [comment]
As of now, the `--vocab_file` argument is no longer needed. You only need specify `--spm_model_file`. More instructions in the readme.
